File: Lab_3/Clothes.java
package ua.lviv.iot;

public class Clothes extends Exhibit{
    private String material;
    
    public Clothes() {
    	
    }
     public Clothes(ExhibitName name, ExhibitAge age, String kindOfArt, String placeOfManufacture, String condition, double weight, String material) {
    	     super(name, age, kindOfArt, placeOfManufacture, condition, weight);
    	     this.material = material;
     }
	public String getMaterial() {
		return material;
	}
	public void setMaterial(String material) {
		this.material = material;
	}
	
	@Override
	public String toString() {
		return super.toString() + ", material: " + this.material  + "\n";
	}
}

File: Python/Lab_2/exhibits/Clothes.py
from exhibits.Exhibit import *
from enums.ExhibitName import *
from enums.ExhibitAge import *


class Clothes(Exhibit):
    exhibit_age = ExhibitAge.SIX_BC
    exhibit_name = ExhibitName.CLOTHES

    def __init__(self, material, name, age, kind_of_art, place_of_manufacture, condition, weight):
        self.material = material
        Exhibit.__init__(self, name, age, kind_of_art, place_of_manufacture, condition, weight)

    def __str__(self):
        return "Name : " + str(self.name) + " age: " + str(self.age) + " the material of clothes is " + str(
            self.material)